packages/cozy-stack-client/src/__tests__/CozyStackClient.spec.js

```diff
@@ -310,6 +314,26 @@ describe('CozyStackClient', () => {
       }
     })
 
+    it('should try to refresh the current token when received an invalid token error', async () => {
+      const client = new CozyStackClient(FAKE_INIT_OPTIONS)
+      global.fetch
+        .mockRejectOnce(
+          new Error(JSON.stringify({ error: 'Invalid JWT token' }))
+        )
+        .once([FAKE_APP_HTML, { status: 200 }])
+        .once([JSON.stringify({ res: 'ok' }), { status: 200 }])
```

Comment:
What is this `once` ?

Reply:
it's from https://github.com/jefflau/jest-fetch-mock#readme

Reply:
OK interesting. I did not think fetch was mocked with a special mock. I personally find `mockResponseOnce` more telling than the `once`, and it is easier to search for it ;)